<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite errors="0" failures="0" hostname="DESKTOP-O2A05I4" name="pytest" skipped="0" tests="5" time="20.528" timestamp="2020-04-16T16:03:23.611130"><testcase classname="TestCases.test_00_login.TestLogin" file="TestCases\test_00_login.py" line="16" name="test_login_user_noUser" time="12.620"></testcase><testcase classname="TestCases.test_00_login.TestLogin" file="TestCases\test_00_login.py" line="21" name="test_wrong_user_passwd" time="1.580"></testcase><testcase classname="TestCases.test_00_login.TestLogin" file="TestCases\test_00_login.py" line="26" name="test_login_noExistUser_wrongPasswd[data0]" time="1.418"></testcase><testcase classname="TestCases.test_00_login.TestLogin" file="TestCases\test_00_login.py" line="26" name="test_login_noExistUser_wrongPasswd[data1]" time="1.293"></testcase><testcase classname="TestCases.test_00_login.TestLogin" file="TestCases\test_00_login.py" line="33" name="test_login_success" time="3.490"></testcase></testsuite></testsuites>